### PR TITLE
Create 1740C.cpp

### PR DESCRIPTION
**Pull request by ryuusama09**

### CODE REVIEW

**Review by Waqar-107 — CHANGES_REQUESTED** [review]
1. Remove the unnecessary template codes. It will be difficult for the reader to find the actual code.
2. The file naming convention is wrong.
3. Fix the indentations

Follow the instructions from here https://github.com/Waqar-107/Codeforces/blob/master/CONTRIBUTING.md

**Comment by ryuusama09** [comment]
link to the accepted solution : https://codeforces.com/contest/1740/submission/178442533
All the required changes have been implemented @Waqar-107

**Review by Waqar-107 — CHANGES_REQUESTED** [review]
1. Indentation has not been FIXED.  There are a lot of blank lines.
2. Remove the commented-out defines
3. For indentation take this code as a reference -> https://github.com/Waqar-107/Codeforces/blob/master/C-set/1551C.%20Interesting%20Story.cpp

**Comment by ryuusama09** [comment]
@Waqar-107 , implemented the required changes . you can review the code and suggest if any further changes are required

File: C-set/1740C-Bricks-and-bags.cpp [hunk 1]
```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
...
-   
+  }
 
+  ll ans = -1;
 
+  for (int i = 0; i < n; i++) {
+    if (i == 0) {
+      ans = max(ans, a[n - 1] + a[1] - 2 * a[0]);
+    }
+    elif(i == n - 1) {
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:63]: this should be `else if`

File: C-set/1740C-Bricks-and-bags.cpp [hunk 2]
```diff
@@ -1,46 +1,34 @@
...
-typedef pair<LL, LL> pll;
-typedef pair<string, string> pss;
-typedef vector<long long int> vi;
-typedef vector<vi> vvi;
-typedef vector<pii> vii;
-typedef vector<LL> vl;
-typedef vector<vl> vvl;
-typedef map<ll,ll> mll;
+typedef long long LL;
+typedef pair < ll, ll > pii;
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:16]: these should be like this -> `typedef pair<ll, ll> pii`. No space required after and before the angle brackets

File: C-set/1740C-Bricks-and-bags.cpp [hunk 3]
```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
...
 
- void solve(){
-   
-   ll n;
-   cin >> n;
-   vi a(n);
-   cin >> a;
+  ll n;
+  cin >> n;
+  vi a(n);
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:47]: add a new line before this

File: C-set/1740C-Bricks-and-bags.cpp [hunk 4]
```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
...
+  cin >> a;
   sort(all(a));
-   set<ll> k;
-   rep(i,n)k.insert(a[i]);
-   if(k.size() == 1){
+  set < ll > k;
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:50]: add a new line before this

File: C-set/1740C-Bricks-and-bags.cpp [hunk 5]
```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
...
+  rep(i, n) k.insert(a[i]);
+  if (k.size() == 1) {
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:52]: add a new line before this

File: C-set/1740C-Bricks-and-bags.cpp [hunk 6]
```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
...
     cout << 0 << '\n';
     return;
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:54]: add a new line before this

Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:54]: add a new line before this

File: C-set/1740C-Bricks-and-bags.cpp (hunk 1 above)
Comment (ryuusama09) [reply C-set/1740C-Bricks-and-bags.cpp:63]: These things have already been mentioned in the defines @Waqar-107

File: C-set/1740C-Bricks-and-bags.cpp [hunk 7]
```diff
@@ -49,177 +37,54 @@ typedef map<ll,ll> mll;
...
-   
+  int T = 1;
+  cin >> T;
+
+  auto start = high_resolution_clock::now();
 
+  for (int t = 1; t <= T; t++) {
+    solve();
+  }
+  return 0;
```
Comment (Waqar-107) [inline C-set/1740C-Bricks-and-bags.cpp:88]: add a new line before this

**Review by Waqar-107 — CHANGES_REQUESTED** [review]
Added a few suggestions

**Comment by ryuusama09** [comment]
Actually @Waqar-107 , you can merge this and implement the required changes , as it would he more sensible and efficient instead of me trying to improve changes everytime 😅

**Pull request closed without merging**